client/blocks/login/error-notice.jsx

```diff
@@ -61,6 +61,11 @@ class ErrorNotice extends Component {
 			return null;
 		}
 
+		// The existing_wpcom_user error is handled inline in LoginForm
```

Comment:
Thanks for the explanation Tug. I was looking for `existing_wpcom_user` in `<LoginForm />` but couldn't find it, which is why without having worked on that code I didn't really find the correlation between this error and the notice you pointed that obvious.